DESCRIPTION

```diff
@@ -46,5 +46,4 @@ LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1.9000
 Language: en-US
-Remotes:
-    tidymodels/rsample
```

Comment:
Did you want to require a specific cran rsample version now? i.e. i dont think it should be `rsample (>= 0.0.8.9001)` anymore, but maybe `rsample (>= 0.0.9)`?

Reply:
I left it as-is to the CI runs won't fail for the next week (for binaries to be built). I can change it later.